src/bitExpert/Adroit/Responder/Responder.php

```diff
@@ -32,7 +32,7 @@
      *
      * @param Payload $payload
      * @param ResponseInterface $response
-     * @return ResponseInterface|null
+     * @return ResponseInterface
      */
     public function __invoke(Payload $payload, ResponseInterface $response);
```

Comment:
If you remove the null option, we also may use return type hint :ResponseInterface for __invoke. If you experience any issues when executing unit tests, please let me know.